EnglishFilter/sections/antiadblock.txt

```diff
@@ -459,11 +462,9 @@ wallpapershome.com###welcome.adblock
 @@||faucetcrypto.com/ads^
 ! https://forum.adguard.com/index.php?threads/38814/
 hentaihaven.xxx#@#.adBanner
-! https://github.com/AdguardTeam/AdguardFilters/issues/57744
-bitlinks.pw#$#img[id^="ads-"] { position: absolute!important; left: -3000px!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/58355
```

Comment:
What about Safari?

Reply:
I suppose that the scriptlet will work in Safari, isn't it?

Reply:
Will work in 1.7.5 - changelog aprove use `set-constant`.

Reply:
Will work in `v1.6.4+` - changelog approve use `set-constant`: 
[AdguardTeam/AdGuardForSafari/releases/tag/v1.6.4](https://github.com/AdguardTeam/AdGuardForSafari/releases/tag/v1.6.4) + https://github.com/AdguardTeam/AdGuardForSafari/issues/237

Reply:
> I suppose that the scriptlet will work in Safari, isn't it?

try JS for macos Safari only.